build error

Sending build context to Docker daemon  55.5 MB
Step 1/16 : FROM golang:1.10.3-stretch as builder
Error parsing reference: "golang:1.10.3-stretch as builder" is not a valid repository/tag: invalid reference format
make: *** [build] Error 1

--- a/Dockerfile
+++ b/Dockerfile
@@ -1,5 +1,5 @@
 # Stage: builder
-FROM golang:1.10.3-stretch as builder
+FROM golang:1.10-stretch as builder
 
 WORKDIR /go/src/github.com/CovenantSQL/CovenantSQL
 COPY . .

